recipe/conda_build_config.yaml

```diff
@@ -6,3 +6,8 @@ cudnn:                                          # [linux64]
   - undefined                                   # [linux64]
 cdt_name:                                       # [linux64]
   - cos7                                        # [linux64]
+
+c_compiler:                                     # [win]
+- vs2019                                        # [win]
```

Comment:
Just to make sure (after adding, reverting, etc.), this now upgrades the windows compilers. I don't mind doing that (actually, even in favour since the death of vs2017 is hastened greatly by msft currently), but just wanted to check if this is now the intention.

Reply:
It's intentional since upstream also uses vs2019.